From 10254fc8216fba30e03c2bb3650d1699bfcb3716 Mon Sep 17 00:00:00 2001
From: Attila Hammer <hammera@pickup.hu>
Date: Fri, 8 Dec 2023 15:53:14 +0100
Subject: [PATCH] tests/Makefile.am: temporary mark xfail with five math_voluve
 tests Following tests produce failures with Liblouis 3.28.0 and Liblouis
 latest master branch version: * test_mathml_woluwe/test_080.test, *
 test_mathml_woluwe/test_106.test, * test_mathml_woluwe/test_107.test, *
 test_mathml_woluwe/test_113.test, * test_mathml_woluwe/test_114.test

Signed-off-by: Attila Hammer <hammera@pickup.hu>
---
 tests/Makefile.am | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index b915a61..2dd9549 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -604,9 +604,14 @@ XFAIL_TESTS +=					\
 	test_mathml_woluwe/test_075.test	\
 	test_mathml_woluwe/test_076.test	\
 	test_mathml_woluwe/test_078.test	\
+	test_mathml_woluwe/test_080.test	\
 	test_mathml_woluwe/test_093.test	\
 	test_mathml_woluwe/test_096.test	\
-	test_mathml_woluwe/test_099.test
+	test_mathml_woluwe/test_099.test	\
+	test_mathml_woluwe/test_106.test	\
+	test_mathml_woluwe/test_107.test	\
+	test_mathml_woluwe/test_113.test	\
+	test_mathml_woluwe/test_114.test
 
 TESTS =						\
 	$(check_PROGRAMS)			\
